index.css

```diff
@@ -0,0 +1,249 @@
+.header {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  width: 100vw;
+  height: 50px;
+}
+
+.my-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+  box-sizing: border-box;
+}
+
+.my-logo > span {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.dropdown {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 50px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.dropbtn {
+  border: none;
+  background-color: none;
+}
+
+.dropbtn img {
+  width: 24px;
+  height: 24px;
+  background-color: none;
+}
+
+/* Dropdown Content (Hidden by Default) */
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f1f1f1;
+  min-width: 160px;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+
+/* Links inside the dropdown */
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover { background-color: #ddd; }
+
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content { display: block; }
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+}
+
+.headline h2 {
+  width: 327px;
+  height: 150px;
+
+  /* Header / 2 */
+
+  font-family: 'Poppins', Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+
+  /* or 130% */
+
+  display: flex;
+  align-items: center;
+
+  /* Neutrals/N800 */
+
+  color: #172b4d;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline p {
+  width: 327px;
+  height: 146px;
+
+  /* Body / 2 */
+
+  font-family: 'Poppins', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* or 150% */
+
+  /* Neutrals/N600 */
+
+  color: #344563;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.headline .Social {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 16px;
+  width: 150px;
```

Comment:
:red_circle: Kindly remove this width property to match the Figma Design. In case you want the width to fit-content kindly use `fit-content` as the width's value 
|Your Design|Figma Design|
|:----------------:|:----------------:|
|![Screenshot from 2023-01-18 00-04-55](https://user-images.githubusercontent.com/26736582/213022628-7e28ba0a-7b9e-47e1-aa64-004476091420.png)|![Screenshot from 2023-01-18 00-05-11](https://user-images.githubusercontent.com/26736582/213022673-427fc8b3-e557-4246-b80a-68cec42a1086.png)
|